python/tests/avro/test_resolver.py

```diff
@@ -280,3 +281,23 @@ class Ints(Record):
             records = list(reader)
 
     assert repr(records) == "[Ints[c=3, d=None]]"
+
+
+def test_resolver_initial_value() -> None:
+    write_schema = Schema(
+        NestedField(1, "name", StringType()),
+        schema_id=1,
+    )
+    read_schema = Schema(
+        NestedField(2, "something", StringType(), required=False, initial_default="vo"),
+        schema_id=1,
```

Comment:
Probably doesn't matter, but this should be incremented.

Reply:
Eagle eye, I've changed it to two.